tests/quantization/test_quantized_ops.py

```diff
@@ -1332,18 +1332,22 @@ def test_batch_normalization(tensor_shape, n_bits, check_r2_score):
 @pytest.mark.parametrize(
     "keepdims", [pytest.param(keepdims, id=f"keepdims-{keepdims}") for keepdims in [0, 1]]
 )
+# In Concrete ML, we consider that all inputs' first dimension should be a batch size
+# even in single batch cases. This is why the following test parameters are considering axes that
+# are sometimes equal to the input size's dimension, as the batch size is added within the
+# test itself.
+# Finally, the axis parameter should neither be None or contain axis 0 as this dimension is used for
```

Comment:
*nor

Reply:
good catch for the typo

as the space above the comment I believe we shouldn't because the convention with decorators (like `@pytest.mark.parametrize`) is to put them directly on top of the function to decorate (not counting comments obviously)